autoware_plugin/include/autoware_plugin.h

```diff
@@ -57,14 +61,34 @@ namespace autoware_plugin
         std::shared_ptr<ros::CARMANodeHandle> nh_, pnh_;
 
         // publisher for CARMA trajectory
-        ros::Publisher  trajectory_pub_;
+        // ros::Publisher  trajectory_pub_;
```

Comment:
If there is old code that is no longer needed after this change you can delete it.

Reply:
deleted